README.md

```diff
@@ -10,49 +10,246 @@ It has only one function: generate the [Fibonacci sequence](https://en.wikipedia
+
+As a first step for publishing to Maven Central, you’ll need to have a verified namespace. The `groupId` of the artifacts you publish will have to start with the name of your verified namespace. For example, if you register the `com.example` namespace, you’ll be able to publish artifacts with the `groupId` set to `com.example` or `com.example.libraryname`.
+
+To get started with publishing to Maven Central, sign in (or create a new account) on the [Maven Central](https://central.sonatype.com/) portal. Once signed in, navigate to [Namespaces](https://central.sonatype.com/publishing/namespaces) under your profile, and click the Add Namespace button. Here, you can register a namespace for your artifacts, either based on your GitHub account or a domain name that you own.
+
+**For a GitHub repository**
+Using your GitHub account to create a namespace is a good option if you don’t own a domain name to use for publication. To create a namespace based on your GitHub account:
+
+1. Enter io.github.`<your username>` as your namespace. For example, `io.github.kotlin-hands-on`.
+2. Copy the Verification Key displayed.
+3. On GitHub, create a new repository with your GitHub account with the verification key as the repository’s name. For example, `http://github.com/kotlin-hands-on/ex4mpl3c0d`.
+4. Navigate back to Maven Central, and click on the Verify Namespace button. After a successful verification, you can delete the repository you’ve created.
+
+**For a domain name**
+To use a domain name that you own as your namespace:
+
+1. Enter your domain as the namespace using reverse-DNS form. If your domain is `example.com`, enter `com.example`.
+2. Copy the Verification Key displayed.
+3. Create a new DNS TXT record with the verification key as its contents. See [Maven Central’s FAQ](https://central.sonatype.org/faq/how-to-set-txt-record/) for more information on how to do this with various domain registrars.
+4. Navigate back to Maven Central, and click on the Verify Namespace button. After a successful verification, you can delete the TXT record you’ve created.
+
+#### Generate a Key Pair
+
+Artifacts published to Maven Central [must be signed with a PGP signature](https://central.sonatype.org/publish/requirements/gpg/), which allows users to validate the origin of artifacts.
+
+To get started with signing, you’ll need to generate a key pair:
+
+* The **private key** is used to sign your artifacts, and should never be shared with others
+* The **public key** can be used by others to validate the signature of the artifacts, and should be published
+
+The `gpg` tool that can manage signatures for you is available from [their website](https://gnupg.org/download/index.html). You can also install it using package managers such as [Homebrew](https://brew.sh/):
+
+| `brew install gpg` |
+| :---- |
+
+Generate a key pair with the following command, and fill in the required details when prompted.
+
+| `gpg --full-generate-key` |
+| :---- |
```

Comment:
It looks strange that these commands are inside of tables... is there a reason for this?

If they're in codeblocks then IntelliJ will add a handy 'run' gutter icon.

<img width="191" alt="image" src="https://github.com/user-attachments/assets/4acf3ee5-5984-4396-b0a2-3e136a8b9d81">

````suggestion
```bash
brew install gpg
```

Generate a key pair with the following command, and fill in the required details when prompted.

```bash
gpg --full-generate-key
```
````